meteor/lib/collections/TimelineDatastore.ts

```diff
@@ -0,0 +1,16 @@
+import { createMongoCollection } from './lib'
+import { registerIndex } from '../database'
+import { TimelineDatastoreEntryId } from '@sofie-automation/corelib/dist/dataModel/Ids'
+export { TimelineDatastoreEntryId }
+import { CollectionName } from '@sofie-automation/corelib/dist/dataModel/Collections'
+
+import { DBTimelineDatastoreEntry } from '@sofie-automation/corelib/dist/dataModel/TimelineDatastore'
+export * from '@sofie-automation/corelib/dist/dataModel/Segment'
```

Comment:
Is this reexport of 'Segment' intentional?

Reply:
No, copy past from Segment.ts. I missed that